pynumaflow/function/async_server.py

```diff
@@ -306,6 +308,17 @@ async def __serve_async(self, server) -> None:
         server.add_insecure_port(self.sock_path)
         _LOGGER.info("GRPC Async Server listening on: %s", self.sock_path)
         await server.start()
+        serv_info = ServerInfo(
+            protocol=info_types.UDS,
+            language=info_types.PYTHON,
+            version=info_server.get_sdk_version(),
+            metadata=info_server.get_metadata_env(info_types.metadata_envs),
+        )
+
+        err = info_server.write(serv_info, info_file=info_types.SERVER_INFO_FILE_PATH)
+        if err is not None:
+            _LOGGER.error(f"Could not write Info-Server {err}")
```

Comment:
Let's use the lazy logging format: https://docs.python.org/3/library/logging.html#logging.debug